### PR TITLE
Allow failure of Python 3.8

### PR DESCRIPTION
**Pull request by expobrain**

### CODE REVIEW

**Comment by isidentical** [comment]
I dont know if it is OK to ask this but an explanation would be really good to understand why allowing failures on 3.8. RC1 is out and soon stable version will be released too.

**Comment by expobrain** [comment]
@isidentical it's absolutely ok to ask!
Python 3.8 is still a release candidate version, not suitable for production, so even if the possibility that this package will be broken under 3.8 it's still possible.
If we didn't allow failures on beta or release candidate version the in case of failures on those version the package will be seen as broken, which actually it's not for stable versions.
Once 3.8.0 is out as a stable version we'll stop allow failures on it.
Hope that this respond to your question; fi you have other questions feel free to ask!

**Comment by isidentical** [comment]
https://mail.python.org/pipermail/python-list/2019-October/743238.html

File: .travis.yml [hunk 1]
```diff
@@ -5,18 +5,18 @@ python:
...
-  # for more information.
-  - python: 3.7
-    sudo: required
-    dist: xenial
-  - python: 3.8-dev
-    sudo: required
-    dist: xenial
+  fast_finish: true
+  allow_failures:
+    - python: 3.8
```
Comment (isidentical) [inline .travis.yml]: > Once 3.8.0 is out as a stable version we'll stop allow failures on it.

so we can remove this, right?

Comment (expobrain) [reply .travis.yml]: Unfortunately not yet, Python 3.8.0 is not yet available on Travis

Comment (isidentical) [reply .travis.yml]: Let's wait https://travis-ci.community/t/add-python-3-8-support/5463

Comment (expobrain) [reply .travis.yml]: +1

Comment (isidentical) [reply .travis.yml]: we're ready to go https://travis-ci.community/t/add-python-3-8-support/5463/6

**Comment by berkerpeksag** [comment]
Thanks!

**Pull request merged**